target_chains/ethereum/contracts/package.json

```diff
@@ -31,8 +31,8 @@
     "@matterlabs/hardhat-zksync-deploy": "^0.6.2",
     "@matterlabs/hardhat-zksync-solc": "^0.3.14",
     "@nomiclabs/hardhat-etherscan": "^3.1.7",
-    "@openzeppelin/contracts": "^4.5.0",
-    "@openzeppelin/contracts-upgradeable": "^4.5.2",
+    "@openzeppelin/contracts": "=4.8.1",
+    "@openzeppelin/contracts-upgradeable": "=4.8.1",
```

Comment:
The contracts might get changed in each upgrade (and actually were). This is the version we were using before the migration to pnpm. The newer versions (although semver compatible) would result in compilation failure.